tbd-fe/src/containers/Form/Form.test.js

```diff
@@ -15,13 +16,6 @@ describe("Form", () => {
     expect(wrapper).toMatchSnapshot();
   })
 
-  it('state should start with empty strings as default', () => {
```

Comment:
Why did we remove these tests?

Reply:
When I pulled down master the constructor for Form was removed so there are no default state values to test